migration-engine/connectors/sql-migration-connector/src/sql_destructive_change_checker/destructive_change_checker_flavour/mssql.rs

```diff
@@ -122,8 +122,10 @@ impl DestructiveChangeCheckerFlavour for MssqlFlavour {
     fn count_rows_in_table<'a>(&'a mut self, table: &'a Table) -> BoxFuture<'a, ConnectorResult<i64>> {
         Box::pin(async move {
             let query = {
-                // TODO(MultiSchema): replace this when implementing MSSQL.
-                let schema_name = self.schema_name();
+                let schema_name = table
+                    .namespace
+                    .clone()
```

Comment:
Do we need to clone here? What if the whatever `namespace` holds could implement `Copy`?

Reply:
It holds an `Option<&str>`, so I think that's a no, unless I'm missing something.